css/_uswds-theme-custom-styles.scss

```diff
@@ -2535,6 +2535,10 @@ $theme-table-column-gap: 4;
         text-indent: 0;
         width: units(3);
       }
+
+      a {
+        text-indent: initial;
+      }
```

Comment:
I see this section uses `usa-content-list`. The external-link mixin could benefit from this change too

https://github.com/uswds/uswds/blob/69cf6929585b5c95ed9700a4502830f84e497e95/packages/uswds-core/src/styles/mixins/general/external-link.scss#L14

Reply:
Opened issue https://github.com/uswds/uswds-site/issues/1960 to address this